File: 2017/problem_b/problem_b.cpp
#include <iostream>
#include <string>
#include <vector>

int main() {
	char days[7][30] = {"Friday", "Saturday", "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday"}; 
	std::vector<std::string> inputs;
	std::string current_line;
	do{
		std::getline(std::cin, current_line);
		inputs.push_back(current_line);
	}
	while(current_line.compare("end") != 0);
	for(int i = 0; i < inputs.size(); i++) {
		int day = 0;
		char text[128];
		if ( sscanf( inputs[i].c_str( ), "%s %d", &text, &day ) != 2 )
			return 0;
		day -= 1;
		std::cout << days[ day % 7] << std::endl;
	}
	return 1;
}

File: 2017/problem_b/problem_b.py
days = ["Friday", "Saturday", "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday"]
inputs = []
eingabe = ""
while True:
    eingabe = input("")
    if eingabe == "end":
        break
    inputs.append(eingabe)
    
for a in inputs:
    asdf = a.split(" ")
    valueinteger = int(asdf[1]) - 1
    print(days[valueinteger % 7])
